Server/mods/deathmatch/logic/packets/CPedWastedPacket.h

```diff
@@ -12,7 +12,6 @@
 #pragma once
 
 #include "CPacket.h"
-#include <CVector.h>
```

Comment:
Are you removing the include because it's already in the precompiled header?

Reply:
yes